docs/getting_started/Demo.md

```diff
@@ -0,0 +1,347 @@
+* Select content type **application/json**
+* Toggle SSL as needed, for testing consider disabling
+* Click **Add webhook**
+
+For more information, consult [GitHub's documentation](https://docs.github.com/en/developers/webhooks-and-events/webhooks/about-webhooks).
+
+#### Update the application source code
+
+In your OpenShift mongo-persistent project page, open the link to the sample application or get the link, by running
+```
+oc get route todolist-route --namespace mongo-persistent -o=go-template='http://{{.spec.host | printf "%s\n" }}'
+```
+
+It should look like in the following image.
+
+![todolist](../img/todolist_orig.png)
+
+The text "Enter an activity" does not seem clear, let's change that to "Add a todo", by running
+```
+sed -i 's/Enter an activity/Add a todo/g' index.html
+```
+
+If you are happy with the change, commit and push
+```
+git add index.html
+git commit -m "update text box"
+git push origin master
+```
+
+Once the commit is pushed to the repository, it will automatically rebuild because we have setup the GitHub webhook.
+
+You will now see that the todolist application starts to rebuild, like in the following image.
+
+![todolist_rebuild](../img/todolist_rebuild.png)
+
+After it finishes, you can now see on your todolist application the updated text change "Add a todo", like in the following image.
+
+![todolist-fixed](../img/todolist_fixed.png)
+
+### Understand the changes to the Grafana Dashboard
+
+Navigate to **"pelorus / Software Delivery Performance - By App"** and set the interval to 15 minutes.
+Pelorus will now read the updated commit and register a new deploytime.  You should now see a total for two deployments, like in the following image.
+
+![First-Update](../img/todolist_update1.png)
+
+The **Lead Time for Change** should initially go down as we just pushed a commit.  The time difference between changes to the original git repository and your personal forked repo will most likely cause this metric to go down.
+
+There have been two deployments since this demonstration was started, the initial deployment and now the redeployment after pushing a change to the git repository. The **Deployment Frequency** should have gone up by 100% in the last 15 minutes.  Once your initial deployment time is longer than 15 minutes in the past, you will find your interval has fallen by 50%.
+
+## Mean Time to Restore and Change Failure Rate
+
+In this section you see how Pelorus measures the last two DORA metrics: **Mean time to recovery** and **Change failure rate**.
+
+![dora_fail_recover](../img/dora_metrics2.png)
+
+It is necessary to have Github issues enabled in your fork of the mig-apps-demo repository (visit settings to enable it).
+
+### Pelorus configuration
+
+After following [Lead Time for Change and Deployment Frequency](#lead-time-for-change-and-deployment-frequency), update the **exporters** section of the `/var/tmp/values.yaml` to match the following example:
+```yaml
+...
```

Comment:
in some yaml files I lefted a `...` to indicate that it is not the whole file. do you think this can confuse the user?